compiler/src/iree/compiler/GlobalOptimization/GeneralizeLinalgNamedOps.cpp

```diff
@@ -30,6 +30,24 @@ struct GeneralizeLinalgNamedOpsPass
 };
 } // namespace
 
+static bool is1x1FilterConv2dOp(linalg::LinalgOp convOp) {
```

Comment:
You should also check that the strides of the conv are 1 (dilations are fine, since they do not affect 1x1 filter).

Also, maybe rename the function to something like `isConvFoldableToContraction`, and add some comments on the function describing the conditions for folding to a contraction.